### PR TITLE
More pod logs docs updates

### PR DESCRIPTION
**Pull request by menehune23**
- Include note about frequent reconnections
- Include clicking as a way to pause

**Targets TAP 1.3**

### CODE REVIEW

**Review by chinigo — APPROVED** [review]
LGTM, ship it!

File: tap-gui/plugins/runtime-resource-visibility.hbs.md [hunk 1]
```diff
@@ -239,3 +239,5 @@ For optimal performance, the pod logs page limits the total log entries download
 
 If the log stream connection is interrupted for any reason, such as a network error, a notification will appear after the most recent log message, and the page will attempt to reconnect to the log stream. If reconnection fails, an error message
 is displayed at the top of the page, and you can click the **Refresh** button at the upper-right of the page to try to reconnect manually.
+
+If you notice frequent disconnections at regular intervals, contact your administrator. Your administrator may need to update the backend configuration for your installation to allow long-lived HTTP connections to log endpoints (endpoints starting with `<backend host>/api/k8s-logging/`).
```
[the remark this answers is not in the document]
Comment (menehune23) [reply tap-gui/plugins/runtime-resource-visibility.hbs.md:243]: I was using the term more generally here, so I was implying both

**Review by obabativavmware — APPROVED** [review]

**Comment by RichardJJG** [comment]
Thanks, @menehune23. I changed `<backend host>` to BACKEND-HOST because that's our placeholder style. If that text is not, in fact, a placeholder, let me know.

**Comment by menehune23** [comment]
> Thanks, @menehune23. I changed `<backend host>` to BACKEND-HOST because that's our placeholder style. If that text is not, in fact, a placeholder, let me know.

Yep, it is a placeholder. Thanks!

**Pull request merged**